src/calculateRentalCost.js

```diff
@@ -29,6 +29,18 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentCostPerDay = 40;
+  const bigRentalCost = 50;
+  const littleRentalCost = 20;
+  let everyDay = days * rentCostPerDay;
+
+  if (days >= 3 && days < 7) {
+    everyDay = everyDay - littleRentalCost;
+  } else if (days >= 7) {
+    everyDay = everyDay - bigRentalCost;
+  }
```

Comment:
previous comment wasn't fixed
```suggestion
  const everyDay = days * rentCostPerDay;

  if (days >= 3 && days < 7) {
    return everyDay - littleRentalCost;
  } else if (days >= 7) {
    return everyDay = everyDay - bigRentalCost;
  }
```